ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -62,13 +67,17 @@
     </LinkedBlock>
   {{/each}}
 {{else}}
-  <EmptyState
-    data-test-secret-list-empty-state
-    @title="No secrets yet"
-    @message="When created, secrets will be listed here. Create a secret to get started."
-  >
-    <LinkTo class="has-top-margin-xs" @route="create">
-      Create secret
-    </LinkTo>
-  </EmptyState>
+  {{#if @filterValue}}
+    <EmptyState @title="There are no secrets matching &quot;{{@filterValue}}&quot;. Press enter to add one." />
```

Comment:
"Please enter to add one" is sort of confusing language. Maybe ask them to clear the filter instead?

Reply:
I was trying to hint at hitting "enter" to add a new secret. But I think that's confusing so I'll just remove the sentence.